GardenApp/app/src/main/java/com/cs1530_group1/gardenapp/Garden.java

```diff
@@ -58,178 +66,642 @@ public String GardenToString()
+    	}
+    	
+        return false;
+    }
+    
+    //done
+    public boolean setPlantDate(String speciesName, Date plantDate)
+    {
+    	Species s = getSpecies(speciesName);
+    	
+    	if (s != null)
+    	{
+        	s.plantDate = plantDate;
+        	return true;
+    	}
+
+    	
+        return false;
+    }
+    
+    //done
+    public boolean setPruneDate(String speciesName, Date pruneDate)
+    {
+    	Species s = getSpecies(speciesName);
+    	
+    	if (s != null)
+    	{
+        	s.pruneDate = pruneDate;
+        	return true;
+    	}
+
+    	
+        return false;
+    }
+    
+    //done
+    public boolean setSunLevel(String speciesName, String sunLevel)
+    {
+    	Species s = getSpecies(speciesName);
+    	
+    	if (s != null)
+    	{
+        	s.sun = sunLevel;
+        	return true;
+    	}
+
+    	
+        return false;
+    }
+    
+    //done
+    public boolean setColor(String speciesName, int color)
+    {
+    	Species s = getSpecies(speciesName);
+    	
+    	if (s != null)
+    	{
+        	s.color = color;
+        	return true;
+    	}
+
+    	
+        return false;
     }
     
+    //done
+    public boolean setSize(String speciesName, int size)
+    {
+    	Species s = getSpecies(speciesName);
+    	
+    	if (s != null)
+    	{
+        	s.size = size;
+        	return true;
+    	}
+
+    	
+        return false;
+    }
+    
+    //done
+    public String getDescription(String speciesName)
+    {
+    	Species s = getSpecies(speciesName);
+    	
+    	if (s != null)
+    	{
+        	return s.des;
+    	}
+    	
+        return null;
+    }
+    
+    //done
+    public String getSpeciesType(String speciesName)
+    {
+    	Species s = getSpecies(speciesName);
+    	
+    	if (s != null)
+    	{
+        	return s.type;
+    	}
+    	
+        return null;
+    }
+    
+    //done
+    public Date getPlantDate(String speciesName)
+    {
+    	Species s = getSpecies(speciesName);
+    	
+    	if (s != null)
+    	{
+        	return s.plantDate;
+    	}
+    	
+        return null;
+    }
+    
+    //done
+    public Date getPruneDate(String speciesName)
+    {
+    	Species s = getSpecies(speciesName);
+    	
+    	if (s != null)
+    	{
+        	return s.pruneDate;
+    	}
+    	
+        return null;
+    }
+    
+    //done
+    public String getSunLevel(String speciesName)
+    {
+    	Species s = getSpecies(speciesName);
+    	
+    	if (s != null)
+    	{
+        	return s.sun;
+    	}
+    	
+        return null;
+    }
+    
+    //done
+    public int getColor(String speciesName)
+    {
+    	Species s = getSpecies(speciesName);
+    	
+    	if (s != null)
+    	{
+        	return s.color;
+    	}
+    	
+        return 0;
+    }
+    
+    //done
+    public int getSize(String speciesName)
+    {
+    	Species s = getSpecies(speciesName);
+    	
+    	if (s != null)
+    	{
+        	return s.size;
+    	}
+    	
+        return 0;
+    }
+    
+    /* Sprint 2 methods */
```

Comment:
keeping all this commented out, unused code in our codebase is pretty ugly, I suggest removing it before it gets merged into the master